README.md

```diff
@@ -1 +1,25 @@
-# opentelemetry-cpp-contrib
+# OpenTelemetry C++ Contrib
+
+This repository contains set of components extending functionality of the
+OpenTelemetry SDK. Instrumentation libraries, exporters, and other components
+can find their home here.
+
+## Contributing
+
+For information on how to contribute, consult [the contributing
+guidelines](./CONTRIBUTING.md)
+
+## Support
+
+This repository is maintained by [.NET SDK
```

Comment:
This repo is **not** maintained by .NET maintainers 😄

Reply:
Sorry, already fixed :)

Reply:
I think as the 1st step we need to clarify who are the maintainers/approvers.